### DISCUSSION TITLE
How can I allow only net requests to one domain?

### DISCUSSION THREAD

**Discussion by tredondo** [netblue30/firejail#4834] [Q&A]
If `net none` is too restrictive, how can we allow only one domain? Is that possible?

I've looked at https://firejail.wordpress.com/features-3/#network (broken anchor link), https://firejail.wordpress.com/documentation-2/building-custom-profiles/, the [FAQ](https://github.com/netblue30/firejail/wiki/Frequently-Asked-Questions), but didn't see anything actionable, and https://firejail.wordpress.com/documentation-2/basic-usage/#networking is super complicated. I just want to allow only one domain in a profile.

**Comment by rusty-snake** [comment]
> If net none is too restrictive, how can we allow only one domain? Is that possible?

No because the internet does not work with domains, it works with IP addresses.

What you can is
- black/whitelist IP addresses with --netfilter
- use [fdns](https://github.com/netblue30/fdns) to filter which domains can be resolved.

**Comment by kmk3** [comment]
@tredondo [on Jan 7](https://github.com/netblue30/firejail/discussions/4834#discussion-3784936):

> If `net none` is too restrictive, how can we allow only one domain? Is that
> possible?

Related:

* #4848
